specification/compute/resource-manager/readme.python.md

```diff
@@ -37,7 +37,7 @@ Generate all API versions currently shipped for this package
 
 ```yaml $(python)
 multiapi: true
-default-api-version: "2024-07-01"
+default-api-version: "2024-03-03"
```

Comment:
We always think the default-api-version shall be same with latest api-version so we shall not change it. @ChenxiJiang333

Reply:
got it